pkg/csi/service/common/common_controller_helper.go

```diff
@@ -141,6 +142,41 @@ func CheckAPI(version string) error {
 	return nil
 }
 
+// UseVslmAPIs checks if specified version is between 6.7.3 and 7.0.0
+// If the version check is ok, the method returns true, else returns false
+// and appropriate errors during failure cases
+func UseVslmAPIs(ctx context.Context, version string) (bool, error) {
```

Comment:
if you are using client.ServiceContent.About.ApiVersion version here, we have 6.7.3, 7.0.0.0, 7.0.1.1 etc.
can you consider validating function with x.x.x.x version scheme as well.

Reply:
Added unit tests to cover these scenarios